content/kubernetes/release-notes/k8s-6-2-18-41-jan-2023.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: Redis Enterprise for Kubernetes release notes 6.2.18-41 (Jan 2023)
+linktitle: 6.2.18-41 (Jan 2023)
+description: This is a maintenance release for 6.2.18 and includes support for Redis Enterprise 6.2.18-72.
+weight: 59
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [ 
+    /kubernetes/release-notes/k8s-6-2-18-41-jan-2023.md,
+    /kubernetes/release-notes/k8s-6-2-18-41-jan-2023/,   ]
+---
+## Overview
+
+This is a maintenance release of Redis Enterprise for Kubernetes 6.2.18-41 that adds supports for Redis Enterprise 6.2.18-72.
```

Comment:
I'm thinking we should include some instructions for users how to consume this, since this isn't a "typical" release - the operator images is the same, so there's no notion of upgrading the operator. RS upgrade needs to be initiated by the user, by updating the RS image tag in its REC spec. Can we have some general entry about this in the upgrade docs, and maybe just refer to it from the release notes?

Reply:
@zcahana I can add some more info to the release notes and link to the upgrade docs. I just need to understand a bit better first: 

The users can skip to the "Upgrade the Redis Enterprise cluster (REC) section of the upgrade guide without having to download and apply a new bundle. Instead they will skip to "Replace the `versionTag:` declaration under `redisEnterpriseImageSpec` with the new version tag."

Is that right? Is this also true for OpenShift customers?

Reply:
@kaitlynmichael yes, this is mostly correct, with a couple of notes:
1. There's no auto-upgrade (`autoUpgradeRedisEnterprise: true`) in this case - only manual upgrade.
2. This is also correct for plain (non-OLM) OpenShift (but note that the [doc](https://docs.redis.com/latest/kubernetes/re-clusters/upgrade-redis-cluster/#edit-redisenterpriseimagespec-in-the-rec-spec) mentions the docker hub image which will be different in OpenShift). 
3. OLM is a different case, in which this does (mostly) behave like a typical operator upgrade - updated operator is first redeployed, after which the REC can be patched to upgrade to the new RS version.